spring-ai-commons/pom.xml

```diff
@@ -61,7 +61,7 @@
 
 		<dependency>
 			<groupId>io.micrometer</groupId>
-			<artifactId>micrometer-tracing</artifactId>
+			<artifactId>micrometer-tracing-bridge-otel</artifactId>
```

Comment:
spring-ai-commons should not depend on a bridge (implementation instead of the abstraction).